runtimes/common/src/constants.rs

```diff
@@ -78,10 +81,24 @@ pub const INFLATION_CONFIG: (Perquintill, Perquintill, Perquintill, Perquintill)
 /// Copied from Kusama & Polkadot runtime
 pub const MAX_VESTING_SCHEDULES: u32 = 28;
 
+/// Calculate the storage deposit based on the number of storage items and the
+/// combined byte size of those items.
+pub const fn deposit(items: u32, bytes: u32) -> Balance {
+	items as Balance * 70 * MILLI_KILT + (bytes as Balance) * 30 * MICRO_KILT
+}
+
+pub const INDICES_SIZE: u32 = 49;
```

Comment:
What does this represent? Probably a comment would help.

Reply:
[dbd748e](https://github.com/KILTprotocol/mashnet-node/pull/329/commits/dbd748e325eedceee15e2ccf42899dc8b7d95422)